Library/Fantasy/Fantasy Advantages.adq

```diff
@@ -19,6 +19,35 @@
 				"points": 2
 			}
 		},
+		{
+			"id": "379c5b53-3fa5-4728-8a3d-be657650cea6",
+			"type": "trait",
+			"name": "Named Possession",
+			"reference": "F132",
+			"tags": [
+				"Perk",
```

Comment:
Perks are defined to be 1 point advantages, just like Quirks are -1 point disadvantages. This tag doesn't seem appropriate here.